intl.emu

```diff
@@ -250,6 +264,29 @@
           1. Remove the last code unit from _m_.
         1. Return the Record { [[FormattedString]]: _m_, [[RoundedNumber]]: _xFinal_, [[IntegerDigitsCount]]: _int_, [[RoundingMagnitude]]: –_f_ }.
       </emu-alg>
+
+      <emu-clause id="sec-decimal128torawfixed" type="abstract operation">
+        <h1>
+          Decimal128ToRawFixed (
+            _x_: a finite Decimal128 value for which sign(_x_) = 1,
+            _minFraction_: an integer in the inclusive interval from 0 to 100,
+            _maxFraction_: an integer in the inclusive interval from 0 to 100,
+            _roundingIncrement_: an integer,
+            _unsignedRoundingMode_: a specification type from the Unsigned Rounding Mode column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref>, or *undefined*,
+          ): a Record with fields [[FormattedString]] (a String), [[RoundedNumber]] (a mathematical value), [[IntegerDigitsCount]] (an integer), and [[RoundingMagnitude]] (an integer)
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>
+            <p>It is the equivalent of <emu-xref href="#sec-torawfixed">ToRawFixed</emu-xref> for Decimal128 values.</p>
+          </dd>
+        </dl>
+        <emu-alg>
+          1. Let _v_ be cohort(_x_).
+          1. If _v_ is *+0*<sub>𝔻</sub>, return ToRawFixed(0, _minFraction_, _maxFraction, _roundingIncrement_, _unsignedRoundingMode_).
+          1. Otherwise, return ToRawFixed(_v_, _minFraction_, _maxFraction, _roundingIncrement_, _unsignedRoundingMode_).
```

Comment:
Issue: You should at some point here pull the precision from the Decimal to retain trailing zeros.

Reply:
(ditto)